### ISSUE TITLE
The loss of boundary is very large and meet the coinverge problem

### ISSUE THREAD

**Issue by jiangxingyuan** [lululxvi/deepxde#430]
Hi Dr. Lu,
I'm trying to utilize deepxde to solve the laser heating problem. Firstly, I want to simplify this problem to a 2D steady state problem with the right, left and bottom boudarys' temperature as 298.13K and the top boundary accept the laser.
The boundary condition of the top boundary is:
![image](https://user-images.githubusercontent.com/87225061/142437107-6491b7f0-b86b-4d79-a024-3aa35bb95eb4.png)
where
![image](https://user-images.githubusercontent.com/87225061/142437221-e145de63-dc7a-4a56-8483-08ca4f027930.png)
Here is my code：

import deepxde as dde
from deepxde.backend import tf
import numpy as np
import scipy.io
import math

def conduction (x,y):

    one  = tf.ones_like(x[0])
    zero  = tf.zeros_like(x[0])
     
    kappa = 60.7*one
    
    dy_xx = dde.grad.hessian(y, x, i=0, j=0)
    dy_yy = dde.grad.hessian(y, x, i=0, j=1)

    e = (dy_xx + dy_yy)
    return e

def T_am(x):
    return 293.15* np.ones((len(x),1), dtype= np.float64)

def func_n(x):
        return np.zeros((len(x),1), dtype=np.float32)
    
def boundary_l(x, on_boundary):
    return on_boundary and np.isclose(x[0], -0.007)

def boundary_r(x, on_boundary):
    return on_boundary and np.isclose(x[0], 0.007)

def boundary_d(x, on_boundary):
    return on_boundary and np.isclose(x[1], 0)

def boundary_u(x, on_boundary):
    return on_boundary and np.isclose(x[1], 0.006)

Tam = 293.15
htcn1 = 100.0
emiss = 0.2
sigm = 5.67e-8  
kappa = 150
 
##### heat loss by convection heat transfer and radiation#################
alaseta = 0.33
alasfact = 1.0 # a 
alaspow = 800 # Q
alasrb = 1.0e-3
scanvel = 0.0
    
geom = dde.geometry.Rectangle([-0.007,0],[0.007,0.006])

bc_l = dde.DirichletBC(geom, T_am, boundary_l)
bc_r = dde.DirichletBC(geom, T_am,  boundary_r )
bc_d = dde.DirichletBC(geom, T_am,  boundary_d)
bc_u = dde.RobinBC(geom, lambda x, y: alasfact*alaspow*alaseta/(math.pi*alasrb**2)*tf.exp(-alasfact*(tf.square(x[:,0:1]))/alasrb**2) - (htcn1*(y - Tam) + emiss*sigm*(y**4 - Tam**4)), boundary_u)

data = dde.data.PDE(geom, conduction, [bc_l, bc_r, bc_d, bc_u],
                                        num_domain=30000,
                                        num_boundary=6000)
net = dde.maps.FNN([2] + [100] * 10 + [1], "tanh", "Glorot uniform")

model = dde.Model(data, net)

model.compile("adam", lr=1e-3)
losshistory, train_state = model.train(epochs=200000)
dde.saveplot(losshistory, train_state, issave=True, isplot=True)
 

This is the loss
![image](https://user-images.githubusercontent.com/87225061/142439188-23782855-df67-4edb-ba3e-082bd5d85acd.png)

![image](https://user-images.githubusercontent.com/87225061/142439431-d02abdfb-937f-4e63-9582-24a4ae3b6284.png)

As you can see, the loss tem of the RobinBC whih corresponding to the top boundary is very large and very hard to converge. The laser power is 800W which lead to the maximum of q_laser could be higher then 10^7. I thought this might make the loss of boundary condition very large.  
Could you give me some advices on how to solve this problem? Thanks a lot.

**Comment by lululxvi** [comment]
See FAQ.

**Comment by lshil00** [comment]
hi, have you solved this problem?
